### PR TITLE
[361] create unit test to categorycontrollerjava _ One file

### PR DESCRIPTION
**Pull request by nataliiapovzhyk**

### CODE REVIEW

File: core/src/test/java/greencity/controller/CategoryControllerTest.java [hunk 1]
```diff
@@ -1,61 +1,86 @@
...
+        long id = 5;
+
+        CategoryDto categoryDto = CategoryDto.builder()
+                .name(name)
+                //.nameUa(nameUa)
+                //.parentCategoryId(id)
+                .build();
+
+        String json = "{\n" +
+                "  \"name\": \""+ name +"\"\n" +
```
Comment (BerserkrMM) [inline core/src/test/java/greencity/controller/CategoryControllerTest.java:60]: better avoid commets

File: core/src/test/java/greencity/controller/CategoryControllerTest.java [hunk 2]
```diff
@@ -1,61 +1,86 @@
...
+                .build();
     }
 
     @Test
+    @DisplayName("Category Controller Post request to create new Category Test")
     void saveCategory() throws Exception {
-        CategoryDto categoryDto = CategoryDto.builder().name("content").build();
+        String name = "Category Name";
+        String nameUa = "Категорія";
+        long id = 5;
```
Comment (BerserkrMM) [inline core/src/test/java/greencity/controller/CategoryControllerTest.java]: nameUa & id is redundant

File: core/src/test/java/greencity/controller/CategoryControllerTest.java (hunk 1 above)
Comment (nataliiapovzhyk) [reply core/src/test/java/greencity/controller/CategoryControllerTest.java:60]: there is a bug with Category DTO mapper that's why these 2 line with setting nameUa & id are not working. So I've commented it to uncomment when bug is fixed

File: core/src/test/java/greencity/controller/CategoryControllerTest.java (hunk 2 above)
Comment (nataliiapovzhyk) [reply core/src/test/java/greencity/controller/CategoryControllerTest.java]: due to a bug with Category DTO mapper these 2 parameters nameUa & id where not used.

File: core/src/test/java/greencity/controller/CategoryControllerTest.java (hunk 1 above)
Comment (nataliiapovzhyk) [reply core/src/test/java/greencity/controller/CategoryControllerTest.java:60]: removed it for now

File: core/src/test/java/greencity/controller/CategoryControllerTest.java (hunk 2 above)
Comment (nataliiapovzhyk) [reply core/src/test/java/greencity/controller/CategoryControllerTest.java]: removed it

**Review by BerserkrMM — APPROVED** [review]
It works.
I would avoid System.out, and extra line breaks like at 40-41,53-54 and 78 lines

**Review by df7734 — APPROVED** [review]

**Review by ushh789 — APPROVED** [review]

**Pull request merged**